Sources/Segment/Utilities/Storage.swift

```diff
@@ -321,10 +321,11 @@ extension Storage {
         let fileEnding = "],\"sentAt\":\"\(sentAt)\",\"writeKey\":\"\(writeKey)\"}"
         do {
             try outputStream.write(fileEnding)
+            try outputStream.close()
```

Comment:
Any chance this should be in a `finally` block?

Reply:
No because it throws and we'd need to have another identical catch in the finally block.